### ISSUE TITLE
Additional standard dialogs

### ISSUE THREAD

**Issue by codebude** [RobinPerris/DarkUI#3]
Hi Robin,
at first - what a great library. When you don't have lots of money (for DevExpress tools, etc.), your library is really a lifesaver! Now my question... Do you plan (or might have some good advices how) to add the standard dialogs: SaveFileDialog, OpenFileDialog, FolderBrowserDialog, etc.?

Most of the controls which aren't part of DarkUI, but standard Winforms controls can easily by "schemed" by setting Fore and Backcolor to match the look'n'feel of DarkUI. Unfortunately the dialogs, named above, aren't that easy to customize. That's why I'm asking for.

**Comment by RobinPerris** [comment]
I looked in to it a while back, and from what I can see that functionality is literally at the operating system level. I couldn't find a way to re-skin them, and all the programs I've used which do have themed file access dialogs would have had to write the entire dialog and file access functionality themselves from scratch.

Even in programs like Visual Studio which have unique themes you're presented with a default Windows file access dialog.

Considering the functionality was already there, and wasn't really part of the main user experience, I didn't think the time commitment was really worth it. It wouldn't take too long to write it yourself if it's important to your project, however. It'd be even quicker if you can find an open-source custom WinForms file access dialog where you can swap the controls to DarkUI versions.

**Comment by codebude** [comment]
Hi Robin,

I understand your points and yes, I remember that I investigated in this topic some time ago. (But unfortunately have forgotten that I did so...).

So from my point it's ok to close this issue, because their won't be a "clean" solution. Maybe I still try it (maybe via native API --> http://www.codeguru.com/csharp/csharp/cs_controls/custom/article.php/c12801/Customizing-OpenFileDialog-in-NET.htm#page-1), but then I would come back to you, if I have success. So for now, let's close the issue.
